lightning/src/ln/channelmanager.rs

```diff
@@ -1551,10 +1553,33 @@ impl ChannelDetails {
 			inbound_htlc_minimum_msat: Some(context.get_holder_htlc_minimum_msat()),
 			inbound_htlc_maximum_msat: context.get_holder_htlc_maximum_msat(),
 			config: Some(context.config()),
+			channel_shutdown_state: Some(context.shutdown_state()),
 		}
 	}
 }
 
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+/// Further information on the details of the channel shutdown.
+/// Upon channels being forced closed, (ie. funding tx spend confirmed detected by chain monitor)
```

Comment:
```suggestion
/// Upon channels being forced closed (i.e. commitment transaction confirmation detected by `ChainMonitor`),
```
since coop close also spends the funding tx

Reply:
thanks for the correction! :)